scripts/utils.js

```diff
@@ -81,3 +81,88 @@ export const sortIconsCompare = (a, b) => {
 		? collator.compare(getIconSlug(a), getIconSlug(b))
 		: collator.compare(a.title, b.title);
 };
+
+/**
+ * Sort icon data obeject.
+ * @param {IconData} icon The icon data as it appears in *_data/simple-icons.json*.
+ * @returns {IconData} The sorted icon data.
+ */
+const sortIcon = (icon) => {
+	const keyOrder = [
+		'title',
+		'slug',
+		'hex',
+		'source',
+		'guidelines',
+		'license',
+		'aliases',
+		// This is not appears in icon data but it's in the alias object.
+		'loc',
+	];
+	const sortedIcon = Object.fromEntries(
+		Object.entries(icon).sort(
+			([key1], [key2]) => keyOrder.indexOf(key1) - keyOrder.indexOf(key2),
+		),
+	);
+	return sortedIcon;
+};
+
+/**
+ * Sort license object.
+ * @param {IconData['license']} license The license object as it appears in *_data/simple-icons.json*.
+ * @returns {IconData['license'] | undefined} The sorted license object.
+ */
+const sortLicense = (license) => {
+	if (!license) return undefined;
+	const keyOrder = ['type', 'url'];
+	const sortedLicense = Object.fromEntries(
+		Object.entries(license).sort(
+			([key1], [key2]) => keyOrder.indexOf(key1) - keyOrder.indexOf(key2),
+		),
+	);
+	return sortedLicense;
+};
+
+/**
+ * Sort object key alphabetically.
+ * @param {IconData['aliases']} object The aliases object as it appears in *_data/simple-icons.json*.
+ * @returns {IconData['aliases'] | undefined} The sorted aliases object.
+ */
+const sortAlphabetically = (object) => {
+	if (!object) return undefined;
+	const sorted = Object.fromEntries(
+		Object.entries(object).sort(([key1], [key2]) => (key1 > key2 ? 1 : -1)),
+	);
+	return sorted;
+};
+
+/**
+ * Sort icons data.
+ * @param {IconData[]} iconsData The icons data as it appears in *_data/simple-icons.json*.
+ * @returns {IconData[]} The sorted icons data.
+ */
+export const formatIconData = (iconsData) => {
+	const mappedIcons = iconsData.map((icon) => {
+		return sortIcon({
+			...icon,
+			license: sortLicense(icon.license),
+			aliases: icon.aliases
+				? sortAlphabetically({
+						aka: icon.aliases.aka?.sort(collator.compare),
+						dup: icon.aliases.dup
+							? icon.aliases.dup.sort(sortIconsCompare).map((d) =>
+									sortIcon({
+										...d,
+										loc: sortAlphabetically(d.loc),
+									}),
+								)
+							: undefined,
+						loc: sortAlphabetically(icon.aliases.loc),
+						old: icon.aliases.old?.sort(collator.compare),
```

Comment:
And remove from here too. Currently we don't have icons with `old` aliases.

Reply:
![](https://github.com/user-attachments/assets/8fb134d6-8b79-4eb2-96c3-53446fb40b81)

There are five icons that have the `old` aliases.

Reply:
Ok. So we are defining a version of the `old` implementation now. I'll open a PR to document it.